src/components/Form/Controls/Toggle.tsx

```diff
@@ -52,17 +51,20 @@ export class Toggle extends Base<Props> {
     return null;
   }
 
-  @Watch('on', { immediate: true})
```

Comment:
Removing that breaks a unit test. I will start to setup the CI now. :) 

<img width="572" alt="screenshot 2019-01-03 at 09 10 59" src="https://user-images.githubusercontent.com/153225/50628244-8a55af80-0f37-11e9-8405-3860c6ae5a75.png">

Reply:
add back the watch block